<ui version="4.0" >
 <class>BrowseReactionsWidgetClass</class>
 <widget class="QDockWidget" name="BrowseReactionsWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>509</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="features" >
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas" >
   <set>Qt::LeftDockWidgetArea</set>
  </property>
  <property name="windowTitle" >
   <string>Reactions</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>559</width>
     <height>487</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="3" >
     <widget class="QTableWidget" name="tableWidget" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="columnCount" >
       <number>5</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="1" column="0" colspan="3" >
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkBox" >
        <property name="text" >
         <string>show selected in dataset window</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="checkBox_2" >
        <property name="text" >
         <string>display metabolite compartments</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2" >
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="title" >
       <string>Information</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Equation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lineEdit" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string> Names</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QListWidget" name="listWidget" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2" >
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="minimumSize" >
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>1500</width>
        <height>100</height>
       </size>
      </property>
      <property name="title" >
       <string>Compound Transitions</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QListWidget" name="listWidget_4" >
         <property name="maximumSize" >
          <size>
           <width>1000</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pushButton_6" >
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text" >
         <string>Show selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton" >
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text" >
         <string>Show all</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
